packages/jsii-calc-base-of-base/package.json

```diff
@@ -42,6 +42,13 @@
           "artifactId": "calculator-base-of-base"
         }
       },
+      "kotlin": {
```

Comment:
Would be nice to add the basic information about this to [`docs/configuration.md`](https://github.com/aws/jsii/blob/master/docs/configuration.md).